Contardo/template/simple_functions.py

```diff
@@ -10,3 +10,9 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(numb):
+	if numb%2:
```

Comment:
this function only check if the input is even.
The command is to return True if the input is prime and False otherwise.
The input must be integer.